backend/routes/auth.py

```
@@ -0,0 +1,59 @@
+    # Get Clerk user ID from frontend
+    auth_header = request.headers.get("Authorization")  # "Bearer <user_id>"
+    if not auth_header:
+        log.error("Missing Authorization header")
+        raise HTTPException(status_code=401, detail="Missing Authorization header")
+
+    try:
+        clerk_user_id = auth_header.replace("Bearer ", "").strip()
+        if not clerk_user_id:
+            log.error("Empty user ID in Authorization header")
+            raise HTTPException(status_code=401, detail="Empty user ID")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)